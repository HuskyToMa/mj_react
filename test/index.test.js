import React from 'react';
import App from '../src/app.jsx';
import renderer from 'react-test-renderer';

test('App component test', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})