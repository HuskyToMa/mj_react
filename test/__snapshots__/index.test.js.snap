// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component test 1`] = `
<div>
  2
</div>
`;
